gen/bbmri_negotiator

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Perun::Agent;
+use Perun::GroupsAgent;
+use open qw/:std :utf8/;
+use Data::Dumper;
+use JSON::XS;
+use Tie::IxHash;
+use utf8;
+
+local $::SERVICE_NAME = "bbmri_negotiator";
+local $::PROTOCOL_VERSION = "1.0.0";
```

Comment:
```suggestion
local $::PROTOCOL_VERSION = "3.0.0";
```
i think they got updated with perun versions in the past and they all got stuck at version 3.0.0